README.md

```diff
@@ -161,28 +161,28 @@ An observation for one agent is a dictionary `{"observation": observation, "acti
+| `army`               | `(N,M)`   | Number of units in a cell regardless of the owner                            |
+| `general`            | `(N,M)`   | Mask indicating cells containing a general                                   |
+| `city`               | `(N,M)`   | Mask indicating cells containing a city                                      |
 | `visible_cells`      | `(N,M)`   | Mask indicating cells that are visible to the agent                          |
-| `owned_cells`        | `(N,M)`   | Mask indicating cells controlled by the agent                                |
+| `owned_cells`        | `(N,M)`   | Mask indicating cells owned by the agent                                     |
 | `opponent_cells`     | `(N,M)`   | Mask indicating cells owned by the opponent                                  |
-| `neutral_cells`      | `(N,M)`   | Mask indicating cells that are not owned by agents                           |
+| `neutral_cells`      | `(N,M)`   | Mask indicating cells that are not owned by any agent                        |
 | `structure`          | `(N,M)`   | Mask indicating whether cells contain cities or mountains, even out of FoV   |
-| `owned_land_count`   |     —     | Number of cells an agent owns                                                |
-| `owned_army_count`   |     —     | Total number of units of an agent over all cells                             |
+| `owned_land_count`   |     —     | Number of cells the agent owns                                               |
+| `owned_army_count`   |     —     | Total number of units owned by the agent                                     |
 | `opponent_land_count`|     —     | Number of cells owned by the opponent                                        |
-| `opponent_army_count`|     —     | Int representing total number of units owned by the opponent                 |
-| `is_winner`          |     —     | Whether agent won                                                            |
-| `timestep`           |     —     | Timestep                                                                     |
+| `opponent_army_count`|     —     | Total number of units owned by the opponent                                  |
+| `is_winner`          |     —     | Indicates whether the agent won                                              |
+| `timestep`           |     —     | Current timestep of the game                                                 |
 
 `action_mask` is a mask with shape `(N,M,4)` where value `[i,j,d]` says whether you can move from cell `[i,j]` in a direction `d`.
    
 ### ⚡ Action
 Action is a `tuple(pass, cell, direction, split)`, where:
 - `pass` indicates whether you want to `1 (pass)` or `0 (play)`.
-- `cell` is an `np.array([i,j])` where `i,j` are indices of the cell you want to move from
+- `cell` is a `np.array([i,j])` where `i,j` are indices of the cell you want to move from
```

Comment:
although I'm not sure, I think 'an' is correct, since it depends on how it sounds, no how its written (?)